### PR TITLE
Changes for running jenkins master from host machine

### PR DESCRIPTION
**Pull request by bamachrn**
* Now jenkins master job will run using codes from host machine
* Fixed scanner provisioning with docker host

### CODE REVIEW

**Comment by bamachrn** [comment]
#dotests

File: jenkinsbuilder/cccp_index_reader.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def projectify(new_project,appid,jobid,giturl,gitpath,gitbranch,targetfile,depen
 
 def main(indexdlocation):
 
-    yamlfile = "./index.yml"
+    yamlfile = "/".join(indexdlocation.split("/")[:-1])+"/index.yml"
```
Comment (rtnpro) [inline jenkinsbuilder/cccp_index_reader.py]: don't do such string manipulation for paths.

Use ``os.path`` module for it: ``os.path.dirname(__file__)``, ``os.path.join``, etc

File: provisions/roles/jenkins/master/tasks/jobs.yml [hunk 2]
```diff
@@ -19,3 +19,16 @@
...
   shell: jenkins-jobs --ignore-cache update {{ ansible_env.HOME}}/index.yml
+
+- name: Ensure /opt/container-pipeline-service/jenkinsbuilder dir exists on Jenkins master
+  file: dest=/opt/container-pipeline-service/jenkinsbuilder state=directory
+
+- name: Get openshift build scripts
+  copy:
+    src: "{{ item }}"
+    dest: /opt/container-pipeline-service/jenkinsbuilder
+  with_items:
```
Comment (rtnpro) [inline provisions/roles/jenkins/master/tasks/jobs.yml]: I think you can copy the entire directory at once

Comment (bamachrn) [inline provisions/roles/jenkins/master/tasks/jobs.yml]: We do not need the whole dir, any way we will move to container based. lets not do the changes now.

**Comment by bamachrn** [comment]
#dotests

File: provisions/roles/jenkins/master/tasks/jobs.yml (hunk 2 above)
Comment (rtnpro) [inline provisions/roles/jenkins/master/tasks/jobs.yml]: OK

**Comment by rtnpro** [comment]
LGTM :+1:

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests

**Pull request merged**